content/release.rst

```diff
@@ -37,6 +37,21 @@ Such a release can have a normal version... and if changes are made
 during the Q&A review a new patch version is released.
 
 
+Release Planning
+================
+
+Tickets marked for the 'next-release' milestone are technically to be
+covered for the next release but in most situations these will fall beyond
+the next release.
+Tickets that must be in the 'next-release' is set with the high priority tag.
+
+The general timeframe between each release is 30 to 45 days.
+However, bigfixes are worked on and released as soon as it is feasible by the
+development team.
+
+We are still currently working on making the release planning process better.
```

Comment:
this is very vague ... better in which direction. what is wrong with the current process :) ?